src/ffi/datetime.rs

```diff
@@ -5,16 +5,29 @@
 //! This is the unsafe thin  wrapper around the [CPython C API](https://docs.python.org/3/c-api/datetime.html),
 //! and covers the various date and time related objects in the Python `datetime`
 //! standard library module.
+//!
+//! (PyPy 7.0.0) - A note regarding PyPy (cpyext) support:
+//!
+//! PyPy's internal representation of dates differs from that of cpython.
```

Comment:
I'm not sure I understand this section or the action that a user is intended to take from it. Using the C API struct *is* supported for PyPy, right? It's just that not all of the functions are available, and you've added one additional variable in its place.

I think the stuff about the import logic and the stuff about the macros in the C API documentation are not relevant here, just keep it simple and say that there is limited function for some of the PyDatetimeCAPI functions in earlier versions of PyPy, and specifically `DateTime_FromTimestamp` and `Date_FromTimestamp` are not currently supported (we can specify the version that they are supported for later, when it comes out).

I'm torn about the `PyDate_FromTimestamp` and `PyDateTime_FromTimestamp` methods. I think don't mention them in this note. Ideally they would go away entirely.

Reply:
I see your point regarding what users would potentially care about.
The original comment was more of a note for of a code comment to help me reason about why the rational of this api.

I've removed most of it and mentioned only `DateTime_FromTimestamp` and `Date_FromTimestamp`.